app/models/chargeback_rate_detail.rb

```diff
@@ -210,6 +210,17 @@ def contiguous_tiers?
     !error
   end
 
+  def affects_metrics
```

Comment:
I'm not sure about the naming of these methods. How about `metrics_keys` and `cost_keys`?